cloudpickle/cloudpickle.py

```diff
@@ -1163,10 +1164,17 @@ class id will also reuse this class definition.
     The "extra" variable is meant to be a dict (or None) that can be used for
     forward compatibility shall the need arise.
     """
-    skeleton_class = type_constructor(name, bases, type_kwargs)
+    skeleton_class = _make_new_class(type_constructor, name, bases, type_kwargs)
     return _lookup_class_or_track(class_tracker_id, skeleton_class)
 
 
+def _make_new_class(type_constructor, name, bases, type_kwargs):
```

Comment:
To avoid unnecessary nesting, I would rather have `type.new_class` be inlined into `_make_skeleton_class`.

Reply:
I do not quite understand this.

Reply:
I mean that `_make_new_class` a wrapper to a function call (`types.new_class`) and nothing else, and used once (in `_make_skeleton_class`). We might as well directly call `types.new_class` inside `_make_skeleton_class`.